pom.xml

```diff
@@ -1227,7 +1227,7 @@
     <ecjVersion>4.4.2</ecjVersion>
     <eclipseGeminiVersion>2.0.0.RELEASE</eclipseGeminiVersion>
     <eclipselinkVersion>2.5.1</eclipselinkVersion>
-    <elasticsearchNettyVersion>4.1.30.Final</elasticsearchNettyVersion>
+    <elasticsearchNettyVersion>4.1.48.Final</elasticsearchNettyVersion>
```

Comment:
This one is only used for testing and should match the version that Elasticsearch is using - no need to update it.